google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplMockitoTest.java

```diff
@@ -16,27 +16,38 @@
 
 package com.google.cloud.storage;
 
+import static com.google.cloud.storage.SignedUrlEncodingHelper.Rfc3986UriEncode;
```

Comment:
IMHO, this isn't familiar enough to static import.

Reply:
done